File: Programs/Intermediate/old_enough.py
#DD Period 7 Old Enough

import sys
try: #I will say, I did ask for HELP from an AI to figure out how to do this part.
    age = int(input("How old are you?\n"))
except ValueError:
    print("AHHHHHHH I'M GONNA EXPLODE!!!")
    sys.exit()

if age >= 50:
    print("Ok, boomer. Your wrinkly hands are dirtying my screen.")

elif age >= 18:
    print("You are old enough to vote. Welcome to politics.")

elif age >= 16:
    print("Congrats! You are old enough to drive!")

elif age >= 15:
    print("Wow! You are old enough to get a learner's permit!")

elif age >= 6:
    print("Hey kiddo, you are old enough to go to school!")

else:
    print("Wut da heck are you doing on a computer?")

File: c_programming/Programs/Easy/old_enough.c
// DD Period 7 Old Enough
#include <stdio.h>
#include <string.h>

int main(void){
    int age;
    printf("Hello, how old are you? I am totally not some internet creep or something.\n");
    scanf("%d", &age);

    if(age>=18){
        printf("You are old enough to vote!\n");
    }
    else if(age>=16){
        printf("You are old enough to drive!\n");
    }
    else if(age>=15){
        printf("You are old enough to get a learners permit!\n");
    }
    else if(age>=5){
        printf("You are old enough to go to school!\n");
    }
    else{
        printf("I aint answering!\n");
    }
    return 0;
}